src/recoil_values/Recoil_selector_NEW.js

```diff
@@ -840,7 +840,6 @@ function selector<T>(
       const message = `Recoil selector has circular dependencies: ${dependencyStack
         .slice(dependencyStack.indexOf(key))
         .join(' \u2192 ')}`;
-      // eslint-disable-next-line fb-www/no-new-error
```

Comment:
this lint rule did not exist for the oss version of recoil, which was causing lint error